static/src/javascripts/lib/fetch-json.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+import config from 'lib/config';
+import fetch from 'lib/fetch';
+
+function json(input: string, init: any = {}) {
```

Comment:
Is it worth defining a [`RequestInit` dictionary](https://fetch.spec.whatwg.org/#requestinit) type here? It seems to be [already done](http://www.saltycrane.com/blog/2016/06/flow-type-cheat-sheet/#lib/dom.js), although under the erroneous name `RequestOptions`.